### DISCUSSION TITLE
How to embed youtube with start time when using youtube extension

### DISCUSSION THREAD

**Discussion by AKRking** [ueberdosis/tiptap#5765] [General Questions] [answered]
for example pasting

https://www.youtube.com/watch?v=vqIfhUzSUbc&t=1147s

will start the video at 1147 seconds

**Comment by pchunky** [comment]
The extension currently doesn't parse the `t` search param in `youtu.be` and `youtube.com/watch` URLs. I've [created a PR](https://github.com/ueberdosis/tiptap/pull/6234) to fix this.

A workaround for now is to either, 
- use the `youtube.com/embed` URL of your video (e.g., `https://www.youtube.com/embed/vqIfhUzSUbc?start=1147`, or
- use the `youtu.be` URL of your video but change the `t` to `start` and remove the `s` (e.g., `https://youtu.be/vqIfhUzSUbc?start=1147`.

The first is recommended as it's an official URL. You can get it from YouTube's share sheet and click "Embed" in the "Share" section, then sniff out the URL from the code snippet. This works because the extension just trusts the URL if it has `/embed` in it and does no parsing.

**Reply by AKRking** [reply]
thats very low scoped process, https://github.com/steven-tey/novel/discussions/499 i tried this, it works but gives hydration error

**Reply by pchunky** [reply, marked as answer]
Hey @AKRking, my PR https://github.com/ueberdosis/tiptap/pull/6234 has been recently merged, so start timestamps should just work now. Cheers!
